### DISCUSSION TITLE
Illustrerende bakgrunn i vær-boksen

### DISCUSSION THREAD

**Discussion by forteller** [entur/tavla#550] [Ideas]
Jeg setter pris på det enkle og rene uttrykket til Tavla, og jeg liker Entur-fargene godt. Men jeg har en følelse av at det kunne vært veldig kult om bakgrunnen på kun vær-boksen kunne illustrert været gjennom å være enten et proft foto eller en illustrasjon som viser været det er snakk om. 

Ihvertfall om man utvider boksen til å være større enn normalt. Nå blir den da bare til en større lilla boks, uten noe ekstra info eller funksjonalitet, men om den kunne blitt et fint værbilde som lå feks øverst eller nederst på tavla, så hadde det både potensielt vært fint og elegant, og det vil formidle informasjonen mye raskere og på lengre avstand, fordi man ikke trenger å lete med øynene etter vær-ikonet og "parse" det inni hodet, man bare skjønner det umiddelbart.

Bare en tanke :)

**Comment by lindtvedtsebastian** [comment]
This discussion is linked to the old (deprecated) version of Tavla and will therefore be closed.
